code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1447,7 +1457,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 
 //APPLICATIONS FUNCTIONS===========================
 
-			if(PDA_APP_RINGER)
+			if("101")//PDA_APP_RINGER
```

Comment:
`if("[PDA_APP_RINGER]")`
Same for the rest

Reply:
I tried that. Something something only constants allowed.

Reply:
I tried using some dynamic constants (wtf?) but nah, won't work.